static/css/style.css

```diff
@@ -508,33 +534,71 @@ div.load-form {
     min-height: 2em;
 }
 
-.fs-remove-button {
-    height: 2.5em;
-    border-radius: 6px;
-    width: 100%;
-    width: -moz-available;
-    width: -webkit-fill-available;
+.save-button {
+    font-size: 15px;
+    border-radius: 9px;
+    height: 3.0em;
+    outline:none;
+    cursor:pointer;
+    background-color: #f3f3f3;
 }
-.fs-remove-button:hover {
+
+.save-button:hover {
+    background-color: #8fb3d0;
```

Comment:
As these are new classes it will just default to what is defined here, I guess if someone decides they need to be slightly different there will need to be a commit made to the foundation repo to alter them.

Reply:
Yes, so we need to be ready to update the foundation repo before the next beta release.

Reply:
What would even need changing about a white button though, the icon inherits the colour of the button icons so that will be successfully altered?

Reply:
Right, that's what we need to check, if all the right colours are inherited without changes or if we need to add something to the foundation css file. In this case the original code uses a different kind of blue that might work with this repo but might look different when merged into the foundation repo.

Reply:
Sure, what is the easiest way to check this?

Reply:
You could clone the foundation repo, configure your repo as a remote repository, and apply your branch to see if something looks out of place. 

You can also get a patch file by adding the extension to the PR URL: https://patch-diff.githubusercontent.com/raw/bbcmicrobit/PythonEditor/pull/155.patch and apply that to a local copy of the foundation repo.

Reply:
I merged my branch into a local copy of the foundation repo and everything seems to look fine to me. Might be worth someone else checking though.

Reply:
We can do one last check after merging into beta-integration in the foundation repo, before deployment.